ci/ray_ci/container.py

```diff
@@ -1,123 +1,69 @@
 import os
 import subprocess
-import sys
 
 from typing import List
 
-import ci.ray_ci.bazel_sharding as bazel_sharding
-
 _DOCKER_ECR_REPO = os.environ.get(
     "RAYCI_WORK_REPO",
     "029272617770.dkr.ecr.us-west-2.amazonaws.com/rayproject/citemp",
 )
+_DOCKER_ENV = [
+    "BUILDKITE_BUILD_URL",
+    "BUILDKITE_BRANCH",
+    "BUILDKITE_COMMIT",
+    "BUILDKITE_JOB_ID",
+    "BUILDKITE_LABEL",
+    "BUILDKITE_PIPELINE_ID",
+]
+_DOCKER_CAP_ADD = [
+    "SYS_PTRACE",
+    "SYS_ADMIN",
+    "NET_ADMIN",
+]
 _RAYCI_BUILD_ID = os.environ.get("RAYCI_BUILD_ID", "unknown")
 
 
-def run_tests(
```

Comment:
all of these functions are moved to tester_container.py, since they are specific to running tests